docs/spec_draft.md

```diff
@@ -769,19 +770,20 @@ Performs element-wise max operation on tensors `lhs` and `rhs` and produces a
 `result` tensor. For floating-point element types, it implements the `maximum`
 operation from the IEEE-754 specification. For complex element type, it performs
 lexicographic comparison on the (real, imaginary) pairs with corner cases TBD.
+For boolean element types, it follows the order `true` > `false`.
```

Comment:
elemental_ir_emitter.cc suggests a more intuitive formulation: `For boolean element types, the behavior is same as
[stablehlo.or](#stablehloor)`.